signing/sign.proj

```diff
@@ -76,7 +76,7 @@
         <Authenticode>Appx</Authenticode>
       </FilesToSign>
     </ItemGroup>
-    <Error Condition="'@(FilesToSign)' == ''" Text="There are no files to sign. FilesToSign group is empty."/>
+    <Error Condition="'@(FilesToSign)' == '' AND '$(TargetRid)' != 'win-arm64'" Text="There are no files to sign. FilesToSign group is empty."/>
```

Comment:
This may be to nitpicky - but can we use the same condition that is in the build?

`'$(Platform)' != 'arm64'`

https://github.com/dotnet/core-setup/blob/f0fed403e0f455ebaf37626a9561b7de5fdbb160/src/pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.builds#L6

Reply:
Changed